.travis.yml

```diff
@@ -1,14 +1,17 @@
 language: node_js
 node_js:
   - '8'
+cache:
+  yarn: true
 cache:
   - node_modules
+before_install:
+  - npm i -g yarn lerna
 install:
-  - npm install
-  - npm install coveralls
-  - lerna bootstrap --no-ci
+  - lerna bootstrap
+  - yarn add coveralls --ignore-workspace-root-check
```

Comment:
Shouldn't it be added to root deps?

Reply:
? 

It is. And it is required ONLY for CI so we are installing in on CI.

Reply:
Let's add it this way. It changes the `yarn.lock` file, so the CI tests won't be predictable. That what I meant. Having another one small dep installed locally isn't such a big problem IMO.

Reply:
What a problem are you trying to solve?

Reply:
Using our `yarn.lock` file. I don't see much sense tracking it by Git and not using it on CI.

Reply:
By removing `--no-ci` flag from Lerna, now it will use the lock file. Still, not related to the issue.

Reply:
`yarn add` will reset the lock file.

Reply:
It's connected to the issue because you're adding the `yarn.lock` file.

Reply:
Ehh. First is being executed `lerna bootstrap` which bootstrap everything and it uses the lock file. Then, when everything is done, we don't care about the deps because they are already installed and we can safely add `coveralls`. If we want to be more precise, we can move the entire `yarn add coveralls --ignore-workspace-root-check` to `after_success` hook. In the future, would be good to be more precise because after 5 hours I finally was able to understand what's the problem (which in fact wasn't a problem).

Reply:
PS: [`yarn.lock`](https://github.com/ckeditor/ckeditor5-dev/blob/master/yarn.lock) was added in https://github.com/ckeditor/ckeditor5-dev/commit/3ba004febaf806ebfae47518c0940a8c22184f11.

Reply:
I meant that installing one package means that you install its dependencies as well, so you can't assume that the `node_modules looks like before as `yarn add` changes `package.json`, `yarn.lock` and `node_modules`. Every installation will affect other packages too. Also, we can't track this way the version of `coveralls` we use. I'm not saying it's a crucial or vulnerable package, but I still don't know why this package can't be simply added to `node_modules`, especially since we always want to use packages at their specific versions.

> PS: yarn.lock  was added in 3ba004f.

Sorry, I overlooked it, GH didn't show removed lines where I scrolled down

<img width="221" alt="Screen Shot 2019-07-05 at 16 04 03" src="https://user-images.githubusercontent.com/2768047/60727905-53d90b80-9f3f-11e9-983f-10b53129e5aa.png">

Reply:
So, can we agree that moving the package to proper Travis hook instead of adding a package that never will be used on our local machines would be a better solution?

Reply:
Yes! 🎉

Reply:
Done.